cli.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'thor'
+require 'exif'
+
+class CLI < Thor
+  desc "scan", "Recursively reads all of the images from the supplied directory"
+  option :path, :type => :string, :required => false, :desc => "Optional parameter that allows any other directory to be passed in."
+
+  def scan
+    report = File.open("report.csv", "w")
```

Comment:
since you are writing a csv, i would recommend using the class `CSV` https://rubyapi.org/o/csv

i think you might be able to do something like:

```rb
CSV.open("report.csv", "w") do |csv|
  csv << ["file name", "GPS data"] # headers

  Dir.glob("**/*").each do |file|
    ...
    csv << [file, data[:gps]]
  end
end
```

Reply:
Yeah, I'll do this in the next PR. The current implementation of the CSV report is more for demo purpose.